docs/source/index.md

```diff
@@ -20,113 +20,183 @@ ExecuTorch provides support for:
 
 * **Strong Model Support** LLMs (Large Language Models),
   CV (Computer Vision), ASR (Automatic Speech Recognition), TTS (Text To Speech)
-* **All Major Platforms** Android, Mac, Linux, Windows
+* **All Major Platforms** Android, iOS, macOS, Linux, Windows, Embedded
 * **Rich Acceleration Support** Apple, Arm, Cadence, MediaTek, NXP, OpenVino, Qualcomm, Vulkan, XNNPACK
 
 ### Documentation Navigation
-#### Introduction
+
```

Comment:
Do you want to list out all items individually? 

How about we just list top-level categories?

I know that it predates you... I really don't like that we have a documentation sitemap on this page to begin with, we used to have something else before.